### PR TITLE
Adds events, cross-domain- and geoscience-libraries

### LINKED ISSUE #53: Add Pangeo EGU sessions / short course as events
We have a new events page on the website - https://www.pangeo.io/events. It would be good to add the planned events at EGU2025 to this page:

Pangeo Session - ESSI2.15   https://meetingorganizer.copernicus.org/EGU25/session/52079 
Pangeo showcase: Geoscience using a Community Driven Approach 
Convener: Tina Odaka
Co-conveners: Anne Fouilloux, Alejandro Coca-Castro, Dean Summers, Ross A. W. Slater 

Pangeo short course - SC 4.14 
https://meetingorganizer.copernicus.org/EGU25/session/53638 
Harnessing the Power of Pangeo: Enhancing Your Scientific Data Analysis Workflow with scalable open source tools 
Convener: Anne Fouilloux
Co-conveners: Tina Odaka, Scott Henderson, Max Jones, Justus Magin 

FAIR & reusability ESSI3.2   https://meetingorganizer.copernicus.org/EGU25/session/52068 
One decade of FAIR Principles:  Data Reusability and Impact in Earth System Sciences 
Convener: Barbara Magagna
Co-conveners: Ivonne Anders, Karsten Peters-von Gehlen, Anne Fouilloux, Jie Dodo Xu 

This would be done by adding a new folder with the event id as the folder name in https://github.com/pangeo-data/pangeo.io/tree/main/src/events, and then adding an index.md file within that folder that contains the event information. The contents would be general markdown but would need the event metadata at the top as in https://raw.githubusercontent.com/pangeo-data/pangeo.io/refs/heads/main/src/events/agu-2024/index.md.

### PR DESCRIPTION
**Pull request by ofk123**
Supposed to close #53,
Adds libraries ref #15 , #16 

EDIT: Fixed wrong issue number reference above

### CODE REVIEW

**Comment by ofk123** [comment]
Changed the name of folders "core-libraries" & "projects" to "crossdomain_libraries" & geoscience_libraries", to use more correct names. But, on second thought I am not sure if that also will change other parts of the code, so let me know if I should revert that..

**Comment by maxrjones** [comment]
thanks, and sorry for my slow reply! I'll take a look at this by the end of the week

**Comment by tinaok** [comment]
@annefou Do we put https://github.com/ofk123/pangeo.io/blob/main/src/events/egu-2025_ESSI_3_2/index.md
under pangeo events in pangeo.io ?

**Comment by ofk123** [comment]
Hi @maxrjones, did you find time to have a look? I will be able to make changes if necessary.

**Comment by maxrjones** [comment]
> Hi @maxrjones, did you find time to have a look? I will be able to make changes if necessary.

Hi @ofk123, I'm deeply sorry for my lack of response here. I got totally swamped with AGU prep and family obligations. I'll have time to take a look tomorrow.

File: src/events/egu-2025_ESSI_3_2/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+endtime: '2025-05-02T17:00+01:00'
+location: 'Vienna, Austria'
+description: 'One decade of FAIR Principles: Data Reusability and Impact in Earth System Sciences.'
+---
+## Overview
+
+- **Pangeo session - ESSI 3.2** [https://meetingorganizer.copernicus.org/EGU25/session/52068](https://meetingorganizer.copernicus.org/EGU25/session/52068)
+- **Convener**: [Barbara Magagna](https://orcid.org/0000-0003-2195-3997)
+- **Co-conveners**:  [Ivonne Anders](https://orcid.org/0000-0001-7337-3009), [Karsten Peters-von Gehlen](https://orcid.org/0000-0003-0158-2957), [Anne Fouilloux](https://discourse.pangeo.io/u/annefou), [Jie Dodo Xu](https://orcid.org/0000-0001-8239-2076)
+- **When**: Sunday, April 27, 2025 (TBA)
```
[the remark this answers is not in the document]
Comment (ofk123) [reply src/events/egu-2025_ESSI_3_2/index.md:13]: Committing these, and will have to update when specific dates are announced.

**Comment by maxrjones** [comment]
Thanks again @ofk123!

**Pull request merged**